webapp/sources/rudder/rudder-web/src/main/elm/rules/tests/ViewRepairedReportsTest.elm

```diff
@@ -0,0 +1,42 @@
+module ViewRepairedReportsTest exposing (..)
```

Comment:
running this test in CI requires changes in a pom.xml somewhere (the one that calls elm-test)

Reply:
Good catch ! Added it in https://github.com/Normation/rudder/pull/5134/commits/92292d513be798047be84c06be9a6ea0b5b5edf5